ui/nuxeo-document-comments/nuxeo-document-comment-thread.js

```diff
@@ -76,15 +76,19 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
               </paper-textarea>
               <dom-if if="[[!_isBlank(text)]]">
                 <template>
-                  <iron-icon
-                    id="submit"
-                    name="submit"
-                    class="main-option opaque"
-                    icon="check"
-                    on-tap="_submitComment"
-                  ></iron-icon>
+                  <button class="comment-iron-icon" on-tap="_toggleDialog" aria-label$="[[label]]">
```

Comment:
Where is this `_toggleDialog` coming from? What about `label `, where is it defined?

Reply:
Perhaps this was just copied from https://github.com/nuxeo/nuxeo-web-ui/commit/0c66615076cedfc186150205ac53135144f9f4f9#diff-0f46fb05d0c2563fd9f39a3decbd101781b809e11953962b54af44cf8f698aefR112?

Reply:
@poonamyadav252  label issue is still not resolved in your commit.